### PR TITLE
docs:  update vim documentation

### PR DESCRIPTION
**Pull request by samsja**
# Context

this PR include some suggestion for the vim plugin documentation of Tabby

### CODE REVIEW

File: clients/vim/README.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
 # Tabby VIM extension
 
+TabbyML is compatible with both Vim and NeoVim text editor via a plugin.
```
Comment (wsxiaoys) [inline clients/vim/README.md:3]:
```suggestion
Tabby is compatible with both Vim and NeoVim text editor via a plugin.
```

File: clients/vim/README.md [hunk 2]
```diff
@@ -18,12 +33,27 @@ Plug 'TabbyML/tabby', {'rtp': 'clients/vim'}
...
+
 ### 📦 Packer and Lazy
+You first need to install either [Packer](https://github.com/wbthomason/packer.nvim) or [Lazy](https://github.com/folke/lazy.nvim).
+
 In this case, you first need to clone the repo in your machine
 ```
 git clone https://github.com/TabbyML/tabby.git ~/tabby
 ```
-Then on the config file:
+You will need to edit your vim config file (`~/.vimrc` for vim and `~/.config/nvim/init.vim` for neovim) and copy paste the following lines in it (between the `plug#begin` and `plug#end` lines)
```
Comment (wsxiaoys) [inline clients/vim/README.md:55]:
```suggestion
You will need to edit `~/.config/nvim/init.vim` for and copy paste the following lines in it.

File: clients/vim/README.md [hunk 3]
```diff
@@ -41,6 +71,21 @@ let g:tabby_server_url = 'http://127.0.0.1:8080'
...
+```
+:Tabby status
+```
+
+You should see
+```
+Tabby is online
+```
+
+If you se `Tabby cannot connect to the server` it means that you need to start the tabbyML server first. Refer to this [documentation](https://tabby.tabbyml.com/docs/installation/)
```
Comment (wsxiaoys) [inline clients/vim/README.md:87]:
```suggestion
If you se `Tabby cannot connect to the server` it means that you need to start the tabby server first. Refer to this [documentation](https://tabby.tabbyml.com/docs/installation/)
```

**Review by wsxiaoys — CHANGES_REQUESTED** [review]

File: clients/vim/README.md (hunk 3 above)
Comment (wsxiaoys) [reply clients/vim/README.md:87]: A few nits

**Review by wsxiaoys — APPROVED** [review]

**Review by icycodes — APPROVED** [review]
Thank you for your excellent documentation!

**Pull request merged**